src/explore-education-statistics-admin/src/pages/release/hooks/useReleaseImageUpload.ts

```diff
@@ -23,9 +23,7 @@ export default function useReleaseImageUpload(
     async (file, updateProgress) => {
       promise.current = releaseImageService
         .upload(releaseId, file, updateProgress)
-        .then(result =>
-          mapValues(result, insertReleaseIdPlaceholders),
-        ) as CancellablePromise<ImageUploadResult>;
+        .then() as CancellablePromise<ImageUploadResult>;
```

Comment:
How come we need this change?

Reply:
If this is left in, you get a broken image immediately after uploading it to a text block - if I remember correctly. So I moved the place where the `{releaseId} placeholders were add to when a block is saved, preventing the user from seeing a broken image.